quickwit/quickwit-config/src/index_config/serialize.rs

```diff
@@ -28,19 +28,21 @@ use crate::{
 };
 
 /// Alias for the latest serialization format.
-type IndexConfigForSerialization = IndexConfigV0_4;
+type IndexConfigForSerialization = IndexConfigV0_5;
 
 #[derive(Clone, Debug, Serialize, Deserialize, utoipa::ToSchema)]
 #[serde(tag = "version")]
 pub(crate) enum VersionedIndexConfig {
-    #[serde(rename = "0.4")]
-    V0_4(IndexConfigV0_4),
+    #[serde(rename = "0.5")]
+    // Retro compatibilty with 0.4.
+    #[serde(rename(deserialize = "0.4"))]
```

Comment:
```suggestion
    #[serde(alias="0.4")]
```

I think alias is what you want here.